src/components/accessories/selectField/SelectField.tsx

```diff
@@ -50,7 +51,7 @@ const SelectField: FunctionComponent<IProps> = ({
         </MenuItem>
         {options.map((option, index) => (
           <MenuItem value={option.value} key={index}>
-            {option.label}
+            {t(option.label)}
```

Comment:
We are not gonna translate locally the api's value. We must communicate with the Back end and pass in some way the app's language.

Reply:
There are some values that we have to translate in ui like Y/N options, male/femele options... I think that we can keep this code at the moment.

Reply:
I understand the purpose behind this but we could improve the code in any case: I don't like that if we're not gonna use the translate function we invoke the function in any case, this could lead to some unexpected behiovurs. Maybe we could optimize this in some way?